engine/swarm/services.md

```diff
@@ -70,6 +70,58 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your local registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+When you don't specify a tag, you get the latest version of an image, because
+Docker appends the `latest` tag. The following two commands are equivalent:
+
+```bash
+$ docker service create --name="myservice" ubuntu
+
+$ docker service create --name="myservice" ubuntu:latest
+```
+
+You can specify any tag which has been pushed to Docker Hub or your private
+registry. For instance, some of the tags available for the `ubuntu` image are
+`latest`, `devel`, `yakkety`, and `16.04`. The image's tag name is resolved to
+the specific digest it points to at the time of service creation. As an
+alternative, you can specify the digest directly, and no digest resolution
+occurs.
+
+After you create a service, its image is never updated unless you explicitly run
+`docker service update` with the `--image` flag as described below. Other update
+operations such as scaling the service, adding or removing networks or volumes,
+renaaming the service, or any other type of update operation do not update the
+service's image.
+
+### Update a service's image after creation
+
+Each tag represents a digest, similar to a Git hash. Some tags, such as
+`latest`, are updated occasionally to point to a new digest. Services are
+constrained to run a specific digest of an image until you update the service
+using `service update` with the `--image` flag, which is supported in Docker
+Engine 1.13 and higher. If you use an older version of Docker Engine, you must
+remove and re-create the service to update its image.
+
+When you run `service update` with the `--image` flag, a background function
```

Comment:
Instead of "background function", let's say "Docker", "the service update operation", or something similar.